src/test/kotlin/no/nav/syfo/identhendelse/IdenthendelseServiceSpek.kt

```diff
@@ -76,6 +77,7 @@ object IdenthendelseServiceSpek : Spek({
                     // Check that arbeidstaker with old/current personident exist in db before update
                     val currentMotedeltakerArbeidstaker = database.getMotedeltakerArbeidstakerByIdent(oldIdent)
                     currentMotedeltakerArbeidstaker.size shouldBeEqualTo 1
+                    val currentUpdatedAt = currentMotedeltakerArbeidstaker.first().updatedAt
```

Comment:
ordet current blir for meg litt ute av kontekst når MotedeltakerArbeidstaker ikke blir med, da leser jeg det som nåværende tidspunkt, ikke tidspunktet til currentMotedeltakerArbeidstaker. Så når man tester lenger ned så leser jeg at vi tester at oppdatertdeltaker sin UpdatedAt er _etter_ nåværende tidspunkt, altså i fremtiden. Hva med feks initialUpdatedAt ?

Reply:
Ja enig, skal få rettet på det 👌🏼